### ISSUE TITLE
Request for Grade

### ISSUE THREAD

**Issue by Lionstein** [Lionstein/animal-rescue-website#2]
For my project, I think my grade should be anywhere from a C to a B. For whats wrong with it, my website isn't aesthetically pretty, the page layout and fonts aren't great, and its missing some big elements to it like some accessibility features. For what I did right, it has a good blend of everything we've learned across the last 3 semesters.  From linking multiple pages and making forms that we learned in first year to dealing with the images that we only learned recently in this semester, it has a large variety across all our webdev classes.

@charlie-quinn Thanks for teaching us, I enjoyed your class :)

**Comment by charlie-quinn** [comment]
✅ `This portion of your final assignment has been graded. A marking rubric has been sent to your college email address.`

⚠️`Reminder: please close this issue once it has been resolved.`
